README.md

```diff
@@ -28,12 +90,39 @@ terraform plan -out=test-plan.tfplan
 Apply infrastructure
 
 ```bash
-terraform apply
+terraform apply -var infrastructure_db_password=<db-password> -var aap_admin_password=<aap-admin-password> -var aap_red_hat_username=<redhat-username> -var aap_red_hat_password=<redhat-password>
 ```
 Confirm to create infrastructure or pass in the `-auto-approve` parameter.
 
+### Installing Red Hat Ansible Automation Platform
+
+At this point you can ssh into one of the controller nodes and run the installer.
```

Comment:
Please give an example command so the user knows how to do this and with which user name:  For example:
ssh azureuser@<controller public IP>

Reply:
I see variable "infrastructure_admin_username" defaults to azureuser, so need to mention this too so they are aware of it.